test/bin.js

```diff
@@ -28,11 +29,13 @@ describe('bin', function() {
       }
       consoleLog.apply(null, arguments)
     }
+    process.exit = function noop() {}
     process.argv = process.argv.slice(0, 2)
   })
 
   afterEach(function() {
     console.log = consoleLog // eslint-disable-line no-console
+    process.exit = processExit
     // purge yargs cache
     delete require.cache[require.resolve('yargs')]
```

Comment:
Respectively, extend the stub with

```JavaScript
  'yargs': {
    '@global': true
  }
```

?

At the end of the day, if it works (and it does), I'm fine with it. Just asked for clarification ...